documentation/modules/oauth/con-oauth-authorization-keycloak-authorization-services.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following module:
+//
+// proc-oauth-authorization-broker-config.adoc
+
+[id='con-oauth-authorization-keycloak-authorization-services_{context}']
+= Manage policies and permissions in Keycloak Authorization Services
+
+== Kafka Authorization Model
+
+The link:https://kafka.apache.org/documentation/#security_authz_primitives[Kafka authorization model] defines resource types, and permissions available for each type.
+Whenever some action is performed by the Kafka client, one or more permissions are checked by the broker (depending on the action performed) using the configured authorizer.
```

Comment:
```suggestion
Whenever an action is performed by the Kafka client, permissions are checked by the broker (depending on the action performed) using the configured authorizer.
```

Reply:
I think I would make this the first sentence...

When an action is performed by a Kafka client on a broker, the broker uses a configured authorizer to check permissions (depending on the action performed). The Kafka authorization model defines resource types, and permissions available for each type.